LoopKit/DosingDecisionStore.swift

```diff
@@ -173,70 +196,106 @@ public struct StoredDosingDecisionData {
     }
 }
 
+public typealias HistoricalGlucoseValue = PredictedGlucoseValue
+
 public struct StoredDosingDecision {
```

Comment:
[nit] consider moving this struct (and it's dependent friend structs) to its own file (just one file is fine)